pkg/cmd/roachtest/tests/cdc.go

```diff
@@ -2058,7 +2058,7 @@ func registerCDC(r registry.Registry) {
 		Owner:            registry.OwnerCDC,
 		Cluster:          r.MakeClusterSpec(7, spec.Geo(), spec.GatherCores(), spec.GCEZones("us-east1-b,us-west1-b")),
 		CompatibleClouds: registry.OnlyGCE,
-		Suites:           registry.Suites(),
+		Suites:           registry.Suites(registry.Nightly),
```

Comment:
Seems like the test might be flaky:
```
2025/09/25 01:13:13 test_runner.go:1293: --- FAIL: cdc/multi-region-execution-locality-tpcc (602.03s)
(assertions.go:369).Fail: 
	Error Trace:	pkg/cmd/roachtest/tests/cdc.go:2097
	            				pkg/cmd/roachtest/test_runner.go:1421
	            				src/runtime/asm_amd64.s:1700
	Error:      	"5" is not less than or equal to "4"
	Test:       	cdc/multi-region-execution-locality-tpcc
	Messages:   	nodeIdx 0 watched 5 spans, total spans watched 9
```

Artifacts from TC run: https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/20516311?buildTab=artifacts#%2Fcdc%2Fmulti-region-execution-locality-tpcc%2Frun_1%2Fdebug.zip;%2Fcdc%2Fmulti-region-execution-locality-tpcc%2Frun_1%2Fartifacts.zip